### DISCUSSION TITLE
Please share your Cura / slicer profile and settings

### DISCUSSION THREAD

**Discussion by kdobrev** [hillsoftware/sv06#11] [Q&A]
First of all, thank you for this wonderful firmware. it is really good.
But since this it is so good I would trust you with cura profile and settings also :) 
I find there are different settings in the Sovol manual, the Cura profile for manual printer profile and the build in profile that comes in Cura 5.3.0
Of course if you are not using Cura, but something else, please share what you have.

Thank you once again for the firware

**Comment by hillsoftware** [comment]
I use Prusa Slicer and don't use Cura.

I just took the Prusa MK3S+ profile and changed retraction to .5 and tweaked my start and ending g-code to my liking.

You can see that here https://github.com/hillsoftware/sv06/discussions/9#discussioncomment-5429068
